scaffolder-templates/register-component/template.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: register-component
+  title: Register existing component to Software Catalog
+  description: Registers existing component (GitHub repository) to Software Catalog
+  tags:
+    - import
+    - catalog
+    - register
+spec:
+  owner: janus-authors
+  system: janus-idp
+  type: service
+
+  parameters:
+    - title: Provide information about the GitHub location
+      required:
+        - githubHost
+        - githubOrganization
+        - repositoryName
+      properties:
+        githubHost:
+          title: GitHub hostname
+          type: string
+          description: Use github.com for GitHub Free, Pro, & Team or specify a hostname of your GitHub Enterprise instance.
+          default: github.com
+        githubOrganization:
+          title: GitHub Organization
+          type: string
+        repositoryName:
+          title: Repository name
+          type: string
+
+    - title: Provide information about the new component
+      required:
+        - componentOwner
+        - componentType
+        - componentLifecycle
+      properties:
+        componentName:
+          title: Component Name
+          type: string
+          description: Name of the created component. If leaved empty the name of the repository will be used.
+        componentOwner:
+          title: Owner
+          description: Select an owner from the list or enter a reference to a Group or a User
+          type: string
+          ui:field: EntityPicker
+          ui:options:
+            catalogFilter:
+              kind:
+                - Group
+                - User
+        componentType:
+          title: Type
+          type: string
+          description: "The type of component. Well-known and common values: service, website, library."
+          default: other
+        componentLifecycle:
+          title: Lifecycle
+          type: string
+          description: "The lifecycle state of the component. Well-known and common values: experimental, production, deprecated."
+          default: unknown
+
+  steps:
+    - action: fetch:plain
+      id: fetchTemplate
+      name: Fetch repository
+      input:
+        url: https://${{ parameters.githubHost }}/${{ parameters.githubOrganization }}/${{ parameters.repositoryName }}
+
+    - id: template
```

Comment:
Please reorder the parameters to be consistent with the other steps. Normally I like to do `id`, `name`, `action`.